pylops/signalprocessing/Interp.py

```diff
@@ -46,28 +46,70 @@ def _linearinterp(M, iava, dims=None, dir=0, dtype='float64'):
     weights = iava - iva_l
 
     # create operators
-    op = Diagonal(1 - weights, dims=dimsd, dir=dir, dtype=dtype) * \
+    Op = Diagonal(1 - weights, dims=dimsd, dir=dir, dtype=dtype) * \
          Restriction(M, iva_l, dims=dims, dir=dir, dtype=dtype) + \
          Diagonal(weights, dims=dimsd, dir=dir, dtype=dtype) * \
          Restriction(M, iva_r, dims=dims, dir=dir, dtype=dtype)
-    return op, iava
+    return Op, iava
+
+
+def _sincinterp(M, iava, dims=None, dir=0, dtype='float64'):
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Redefining built-in 'dir'](https://app.codacy.com/manual/mrava87/pylops/pullRequest?prid=5328935)